charts/simple-app/values.yaml

```diff
@@ -110,17 +110,11 @@ startupProbe:
 # livenessProbe will be applied to the proxySidecar container instead if that
 # is enabled.
 livenessProbe:
-  httpGet:
```

Comment:
I don't think this is the answer we want... there are far more stacks that _use this_ than that do _not_ use this. Without a default liveness/readiness probe, we're relying on engineers to figureo ut on their own that they need these.

Please do not merge this for now. This will require internal discussion.

Reply:
> we're relying on engineers to figureo ut on their own that they need these.

@tsingla1998's point is that these two are critical and it should require developers to think and be very explicit about how liveness and readiness is treated for their application.

Reply:
So - I could imagine a scenario where we make the chart _hard fail_ if you don't put in a `livenessProbe` and `readinessProbe`? But having them simply _not there_ by default doesn't feel right to me. What do you think about that?

Reply:
Agreed and done.